app-emulation/vagrant/vagrant-2.2.14.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( app-emulation/virtualbox )"
+
+ruby_add_rdepend "
+	>=dev-ruby/bcrypt_pbkdf-1.0.0
+	>=dev-ruby/childprocess-4.0.0
+	>=dev-ruby/ed25519-1.2.4
+	dev-ruby/erubi
+	>=dev-ruby/hashicorp-checkpoint-0.1.5
+	>=dev-ruby/i18n-1.8:1
+	>=dev-ruby/listen-3.1
+	<dev-ruby/log4r-1.1.11
+	>=dev-ruby/mime-types-3.3:*
+	>=dev-ruby/rubyzip-2.0
+	>=dev-ruby/net-scp-3.0.0
+	>=dev-ruby/net-sftp-3.0
+	>=dev-ruby/net-ssh-6.2.0-r1:*
+	dev-ruby/rest-client:2
+	>=dev-ruby/vagrant_cloud-3.0.2
+"
+
+ruby_add_bdepend "
+	>=dev-ruby/rake-12.3.3
+	test? (
+		dev-ruby/rspec
+		dev-ruby/rspec-its
+		dev-ruby/webmock
+	)
+"
+
+all_ruby_prepare() {
+	# remove bundler support
+	sed -i '/[Bb]undler/d' Rakefile || die
+	rm tasks/bundler.rake || die
+
+	sed -e 's/..\/vagrant-spec\|..\/..\/vagrant-spec/\/home\/gseren\/tmp\/vagrant-spec/' \
```

Comment:
This is very hard to read. You can use a different delimiter for `s` to make this more readable, e.g. `s:/path:path:`. I'm also not sure why there is a /home/gseren path there?

Reply:
Ah yes this was part of the my testing branch shouldn't had been there.

On this subject I need your advice @graaff, vagrant need the vagrant-spec package to be able to run its tests,
the thing is vagrant-spec does not ship any release (https://github.com/hashicorp/vagrant-spec).
So I did try to make it work, with git-r3 eclass, but didn't end with something clean maybe I did something wrong.

What do you think about me packaging the vagrant-spec 'snapshoted' on a date (same as vagrant release as it should related) ?

On the technical implementation, I will cleanup my other branch since I learned far more from the review on the eclass, add it to this branch (if you aggree) and we can talk about it.